### PR TITLE
Initial commit to build a representation of the EOSC PROVIDER Profile.

### PR DESCRIPTION
**Pull request by ACz-UniBi**
Initial commit to build the documentation for the EOSC PROVIDER Profile and make an HTML representation of the Profile schema.

Preview of the presentation available at https://draft-eosc-platform-provider-profile.readthedocs.io/en/latest/

### CODE REVIEW

**Comment by kkoumantaros** [comment]
I'm a bit confused by the Versions item in the table of contents, otherwise it looks fine to me.

**Comment by ACz-UniBi** [comment]
Dear @kkoumantaros , thank you for reviewing. Could you refer to the file and line you mean regarding "confused by the Versions item in the table of contents"?

Under "Files changed" you can comment on each line if you wish.

File: docs/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Resources to the EOSC. EOSC Providers are organisations, a part of an organisation or a
+federation that manages and delivers Resources to End-Users. EOSC Providers can be:
+Resource Providers, Service Providers, Data (Source) Providers, Service Developers,
+Research Infrastructures, Distributed Research Infrastructures, Resource Aggregators,
+Thematic Clouds, Regional Clouds, etc.
+
+More definitions `here <https://confluence.egi.eu/display/EOSCEN/EOSC+Definitions>`_.
+(Source: https://wiki.eoscfuture.eu/display/PUBLIC/A.+v4.00+EOSC+Provider+Profile )
+
+Versions
```
Comment (kkoumantaros) [inline docs/introduction.rst:14]: I would remove this section as this doc refers to v4.0 only
alternatively we could change this to change log we keep in github ?

Comment (ACz-UniBi) [reply docs/introduction.rst:14]: Agreed, to remove this before we generate the HTML representation.

**Pull request merged**